service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/ActionCaseManagementOrderControllerSubmittedTest.java

```diff
@@ -98,96 +100,54 @@ void setUp() {
     }
 
     @Test
-    void shouldTriggerCMOProgressionEventAndSendCaseLinkNotificationsWhenIssuedOrderApproved()
+    void shouldTriggerCMOProgressionEventAndNotifyRelevantPartiesWhenCMOIssued()
         throws Exception {
-        List<Element<Representative>> representativesServedByDigitalService =
-            buildRepresentativesServedByDigitalService();
 
-        CaseDetails caseDetails =
-            populateRepresentativesByServedPreferenceData(representativesServedByDigitalService);
+        CaseDetails caseDetails = populateRepresentativesByServedPreferenceData(buildRepresentatives());
 
         postSubmittedEvent(caseDetails);
 
-        verifyCMOTriggerEventsAndNotificationSentToLocalAuthorityOnApprovedCMO();
-        verifySentDocumentEventTriggered();
+        verify(coreCaseDataService).triggerEvent(JURISDICTION, CASE_TYPE, 12345L, CMO_EVENT_KEY);
 
-        verify(notificationClient).sendEmail(
-            CMO_ORDER_ISSUED_CASE_LINK_NOTIFICATION_TEMPLATE, "abc@example.com",
-            getExpectedCMOIssuedCaseLinkNotificationParameters("Jon Snow"), CASE_ID);
-
-        verify(notificationClient).sendEmail(
-            CMO_ORDER_ISSUED_CASE_LINK_NOTIFICATION_TEMPLATE, "xyz@example.com",
-            getExpectedCMOIssuedCaseLinkNotificationParameters("Hodo"), CASE_ID);
-
-        verify(notificationClient).sendEmail(
-            eq(CMO_ORDER_ISSUED_DOCUMENT_LINK_NOTIFICATION_TEMPLATE), eq(CAFCASS_EMAIL_ADDRESS),
-            anyMap(), eq(CASE_ID));
-
-        verifyNotificationSentToAdminWhenCMOIssuedWithNoServingNeeded();
-
-        verifyZeroInteractions(notificationClient);
-    }
-
-    @Test
-    void shouldTriggerCMOProgressionEventAndSendDocumentLinkNotificationsWhenIssuedOrderApproved()
-        throws Exception {
-        List<Element<Representative>> representativesServedByEmail = buildRepresentativesServedByEmail();
-
-        CaseDetails caseDetails = populateRepresentativesByServedPreferenceData(representativesServedByEmail);
-
-        postSubmittedEvent(caseDetails);
-
-        verifyCMOTriggerEventsAndNotificationSentToLocalAuthorityOnApprovedCMO();
-        verifySentDocumentEventTriggered();
+        verify(coreCaseDataService).triggerEvent(JURISDICTION, CASE_TYPE, 12345L, SEND_DOCUMENT_KEY,
+            Map.of("documentToBeSent", CMO_DOCUMENT));
 
         verify(notificationClient).sendEmail(
-            eq(CMO_ORDER_ISSUED_DOCUMENT_LINK_NOTIFICATION_TEMPLATE), eq(CAFCASS_EMAIL_ADDRESS),
-            anyMap(), eq(CASE_ID));
+            CMO_ORDER_ISSUED_CASE_LINK_NOTIFICATION_TEMPLATE,
+            LOCAL_AUTHORITY_EMAIL_ADDRESS,
+            getExpectedCMOIssuedCaseUrlParameters(LOCAL_AUTHORITY_NAME),
+            CASE_ID);
 
         verify(notificationClient).sendEmail(
-            eq(CMO_ORDER_ISSUED_DOCUMENT_LINK_NOTIFICATION_TEMPLATE), eq("jamie@example.com"),
-            anyMap(), eq(CASE_ID));
+            CMO_ORDER_ISSUED_CASE_LINK_NOTIFICATION_TEMPLATE,
+            "abc@digitalrep.com",
```

Comment:
minor: If we have constant for it as for admin or la then it would be super easy to understands to whom email is sent

Reply:
agreed + added couple more constants 👍